deploy.sh

```diff
@@ -20,7 +20,7 @@ dapp --use solc:0.6.12 build
 
 echo "Deploying contracts..."
 
-# Deploy AuthGemJoin5
+# Deploy Gem Join
```

Comment:
does this need to be `Deploy AuthGemJoin8`?

Reply:
No, it was just a typo I noticed from before. This script deploys any of the gem joins.